Turbolinks/WebView.swift

```diff
@@ -40,6 +40,11 @@ class WebView: WKWebView {
         if #available(iOS 11, *) {
             scrollView.contentInsetAdjustmentBehavior = .never
         }
+        #if DEBUG
+          if #available(iOS 16.4, *) {
+            isInspectable = true
+          }
+        #endif
```

Comment:
Yea so starting with iOS 16.4, the webview is no longer inspectable using Safari by default, so we have to explicitly enable it in debug builds of the app for local dev purposes. The `if #available(iOS 16.4, *)` is needed because we target iOS 14 as a deployment target, so the compiler will complain about the `isInspectable` line unless it is wrapper in that 16.4+ qualifier (it was [added](https://developer.apple.com/documentation/webkit/wkwebview/4111163-isinspectable) with SDK 16.4)

Reply:
👍 Thanks, make sense. Another thing Apple breaks with little heads-up 😅